src/ResultList.php

```diff
@@ -82,30 +95,30 @@ public function getIDs()
     }
 
     /**
-     * @return \Elastica\Index
+     * @return Index
      */
     public function getIndex()
     {
         return $this->index;
     }
 
     /**
-     * @return \Elastica\Query
+     * @return Query
      */
     public function getQuery()
     {
         return $this->query;
     }
 
     /**
-     * @return array|\Elastica\ResultSet
+     * @return array|ResultSet
      */
     public function getResults()
     {
         if (is_null($this->resultSet)) {
             try {
                 $this->resultSet = $this->index->search($this->query);
-            } catch (\Exception $e) {
+            } catch (Exception $e) {
```

Comment:
See previous comment about dedicated exception - optional

Reply:
Might leave this for later, seperate from the upgrade work.